ratpack-dropwizard-metrics/src/main/java/ratpack/dropwizard/metrics/DropwizardMetricsModule.java

```diff
@@ -240,6 +241,18 @@ public void onStart(StartEvent event) throws Exception {
         metricRegistry.registerAll(new ThreadStatesGaugeSet());
         metricRegistry.registerAll(new MemoryUsageGaugeSet());
       }
+
+      if (config.isByteBufAllocatorMetrics()) {
+        final MetricRegistry metricRegistry = injector.getInstance(MetricRegistry.class);
+        // Ratpack uses default PooledByteBufAllocator to allocate request bodies and other things.
+        // Allocator is resolved in static block in PooledByteBufAllocator based on OS, presence of Unsafe package, etc.
+        metricRegistry.registerAll(
+          new PooledByteBufAllocatorMetricSet(
+            PooledByteBufAllocator.DEFAULT,
```

Comment:
For some parts of Ratpack, the `ByteBufAllocator` is overridable in the Registry - https://github.com/ratpack/ratpack/blob/master/ratpack-core/src/main/java/ratpack/server/internal/ServerRegistry.java#L98

For example: https://github.com/ratpack/ratpack/blob/master/ratpack-core/src/main/java/ratpack/websocket/WebSockets.java#L77

So, we should probably configure metrics collection for both the `DEFAULT` and the `ByteBufAllocator` from the registry, which you can obtain from the `StartEvent`